installer/mac/copy_x64_binary.py

```diff
@@ -22,10 +22,19 @@ def copy_x64(x64_src_path, x64_dest_path):
     if (os.path.exists(x64_dest_path)):
         shutil.rmtree(x64_dest_path)
 
+    if brave_channel == 'release':
```

Comment:
just for future reference, the logic for mapping channel should not be duplicated in multiple places. It would have been better to pass in the correct value here from gn, but we should just add brave_resources.pak to the ignore list here https://github.com/brave/brave-core/pull/7479/commits/16ac5f13e8e75d68d74241edc2b3aaf92c37b5d7#diff-505cde5aec30cb5bad77c697e87d2db777914873135cc260b247c8275d1014daR31